texturecache.py

```diff
@@ -2301,7 +2301,7 @@ def jsonWaitForCleanFinished(self, id, method, params):
         if title:
           self.logger.out("Cleaning library: %-9s %5d [%s]\n" % (iType + "id", libraryId, title))
         else:
-          self.logger.out("Cleaning library: %-9s %5d [%s]\n" % (iType + "id", libraryId, title))
+          self.logger.out("Cleaning library: %-9s %5d\n" % (iType + "id", libraryId))
```

Comment:
@MilhouseVH 

This is what your script was before, somehow my chnages slipped in.

When I said my eyes were coned shaped it was no joke :tongue: